client/my-sites/pages/page/index.js

```diff
@@ -670,6 +682,25 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Copy Page' );
 	};
 
+	exportPage = () => {
+		this.props.recordEvent( 'Clicked Export Page' );
+		const { page } = this.props;
+
+		const fileContent = JSON.stringify( {
+			__file: 'wp_template',
+			language: 'en',
+			title: page.title,
```

Comment:
Just ensure this works even if the title is empty — I didn't look if `page` object itself always has fallback or if you should provide here empty string `|| ''`